app/policies/induction_coordinator_profile_policy.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+class InductionCoordinatorProfilePolicy < ApplicationPolicy
```

Comment:
Are we missing deleting induction coordinators in controllers and views?

Reply:
Deleted